src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -25,4 +43,35 @@ public SearchIndexModel indexSnapshot(Snapshot snapshot, SearchIndexRequest sear
         List<Map<String, Object>> values = bigQueryPdao.getSnapshotTableData(snapshot, searchIndexRequest.getSql());
         return new SearchIndexModel();
     }
+    public SearchQueryResultModel querySnapshot(
+            SearchQueryRequest searchQueryRequest, List<String> indicesToQuery,
+            @Valid Integer offset, @Valid Integer limit
+    ) {
+        //todo: move to bean for configuration, make injectable
+        final RestClientBuilder builder = RestClient.builder( new HttpHost("localhost", 9200));
+        try (RestHighLevelClient client = new RestHighLevelClient(builder)) {
```

Comment:
You may want to check out what I did in my PR, I pulled the `RestHighLevelClient` into the `SearchService` constructor and let it get built automatically. I'm not sure which way is the most ideal.

Reply:
I think neither, we want it to be constructed outside the class and injected anyways